### PR TITLE
New Layout - UX/UI changes

### PR DESCRIPTION
**Pull request by fmelo**
New styles began to be added. To make them editable as much as possible, color and font variables were created in a new CSS sheet called `template.css`

**You will see things like:**
```
:root {
  --primary-color: #572990;
  --gray-soft: #DFDFDF;
  --primary-font: 'Lato', sans-serif;
}

```
The first page that was changed is: "Your domains", in which, markup, and styles were changed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -219,8 +219,8 @@
     "web3": "^0.20.7",
     "web3-eth-abi": "^1.2.6",
     "web3-stream-provider": "^4.0.0",
-    "web3-utils": "^1.2.4",
-    "xtend": "^4.0.1"
+    "xtend": "^4.0.1",
```
Comment (marcosmartinez7) [inline package.json]: I think this must not be commited on this PR right? @JONAF2103

File: rif.config.js [hunk 2]
```diff
@@ -10,18 +10,18 @@ module.exports = {
...
-      publicResolver: '',
-      nameResolver: '',
-      multiChainResolver: '',
-      rif: '',
-      fifsRegistrar: '',
-      fifsAddrRegistrar: '',
-      rskOwner: '',
-      renewer: '',
-      stringResolver: '',
+      rns: '0xDA7Ce79725418F4F6E13Bf5F520C89Cec5f6A974',
```
Comment (marcosmartinez7) [inline rif.config.js]: This file not be commited too

**Review by marcosmartinez7 — CHANGES_REQUESTED** [review]

File: ui/app/rif/pages/domainsPage/domainsPage.js [hunk 3]
```diff
@@ -40,27 +40,39 @@ class DomainsScreen extends Component {
...
         </div>
-        <div id="chipletDescription" className={'chiplet-description'}>
-          <div id="chipletExpiration">
-            <span>Expires on: {data.details ? data.details.expiration : 'n/a'}</span>
-          </div>
-          <div id="chipletRenew">
-            <span>Auto-renew: <a href={data.details ? this.props.setAutoRenew() : () => {}}>{data.details ? (data.details.autoRenew ? 'on' : 'off') : 'n/a'}</a></span>
-          </div>
-        </div>
+        {/*<div id="chipletDescription" className={'chiplet-description'}>*/}
```
Comment (rorolopetegui) [inline ui/app/rif/pages/domainsPage/domainsPage.js:44]: I think before merging this PR we should delete this comments

File: package.json (hunk 1 above)
Comment (rorolopetegui) [reply package.json]: We are using web3-utils in some lumino classes, or this will be changed?

**Review by rorolopetegui — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (JONAF2103) [reply package.json]: Yup this is a typo probably

File: rif.config.js (hunk 2 above)
Comment (JONAF2103) [reply rif.config.js]: I told him we can remove it later but if you want we can do it here, anyway this file will be overwritten by gulp so i think it doesn't matter, anyway to keep the line of work i think you can remove the changes here @fmelo, you can do that by reseting this file with git. We should have this file in the .gitignore file anyway

File: package.json (hunk 1 above)
Comment (fmelo) [reply package.json]: This line was added because, after the merge, "xtend" was deleted, but, if it is not necessary I can remove it.

Comment (JONAF2103) [reply package.json]: Fixed @fmelo  no worries

File: rif.config.js (hunk 2 above)
Comment (JONAF2103) [reply rif.config.js]: Same here guys, fixed

File: ui/app/rif/pages/domainsPage/domainsPage.js (hunk 3 above)
Comment (JONAF2103) [reply ui/app/rif/pages/domainsPage/domainsPage.js:44]: I think is just a change on the designs, that's why he commented that out, maybe we can remove that but i don't know

Comment (fmelo) [reply ui/app/rif/pages/domainsPage/domainsPage.js:44]: After talking with @anarancio,  we conclude that we should keep that for now.

Comment (fmelo) [reply ui/app/rif/pages/domainsPage/domainsPage.js:44]: Done

**Review by marcosmartinez7 — APPROVED** [review]
lgtm

**Review by JONAF2103 — APPROVED** [review]
LGTM

**Pull request merged**